modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMFormFieldValueUtil.java

```diff
@@ -119,7 +120,13 @@ public static JSONArray getOptionsValuesJSONArray(
 			return createJSONArray("[]");
 		}
 
-		return createJSONArray(value.getString(locale));
+		String valueString = value.getString(locale);
```

Comment:
I'm worried about `value.getString(locale)` because there is a risk of returning null. Can we modify to `GetterUtil.getString(value.getString(locale)) `?

Reply:
Done!